spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -789,9 +789,9 @@ def method
 
       def method
         # comment 1
+
```

Comment:
Is there any reason for the changes in this file?

Reply:
This newline added by auto-correction of `Style/CommentedKeyword` has been added due to a collision with auto-correction of `Style/RedandantBegin`. A solution hasn't come up yet 💦